test/jest/coverage.js

```diff
@@ -47,62 +49,70 @@ function traceAPICoverage(apiCoverage, events, className, classType) {
         apiCoverage.set(`${className}.emit(${JSON.stringify(event)})`, true);
       return method.call(this, event, ...args);
     });
+    uninstalls.push(() => Reflect.set(classType.prototype, 'emit', method));
   }
+
+  return () => uninstalls.forEach(u => u());
 }
 
-describe.skip(!process.env.COVERAGE)('**API COVERAGE**', () => {
+/**
+ * @param {string} browserName 
+ */
+function apiForBrowser(browserName) {
   const BROWSER_CONFIGS = [
     {
       name: 'Firefox',
-      events: require('../lib/events').Events,
+      events: require('../../lib/events').Events,
       missingCoverage: ['cDPSession.send', 'cDPSession.detach'],
```

Comment:
I don't see missingCoverage being used...

Reply:
Yes, deleted it. I got rid of missing coverage here as there were already cases where it wasn't covering all edge cases. Edge cases are all in one place in checkCoverage.js.